scanner/src/funTest/kotlin/scanners/AbstractScannerFunTest.kt

```diff
@@ -63,7 +63,7 @@ abstract class AbstractScannerFunTest(testTags: Set<Tag> = emptySet()) : StringS
             val summary = result.scanner?.results?.scanResults?.singleOrNull()?.singleOrNull()?.summary
 
             summary shouldNotBeNull {
-                licenses shouldBe expectedFileLicenses
+                licenseFindings shouldContainInOrder expectedFileLicenses
```

Comment:
This should be `shouldContainExactlyInAnyOrder`.

Reply:
Thanks for catching this, fixed.